test/e2e/docs/miscellaneous.md

```diff
@@ -0,0 +1,65 @@
+# Other Information
+
+## Table of Contents
+
+- [NodeJS Version](#nodejs-version)
+- [Git Pre-Commit Hook](#git-pre-commit-hook)
+- [Launch Logged-In Window](#launch-logged-in-window)
+- [User account requirements](#user-account-requirements)
+- [List of wrapper repos & friends](#list-of-wrapper-repos--friends)
+- [What to name your branch](#what-to-name-your-branch)
+
+## NodeJS Version
+
+The node version should be defined in the `.nvmrc` file for use with the [nvm](https://github.com/creationix/nvm) project.
+
+## Git Pre-Commit Hook
+
+The file `/scripts/git-pre-commit-circleci-validate` will run `circleci validate` against the CircleCI config file prior to every commit.  This prevents the constant back-and-forth when making updates only to find that they fail immediately on CI.  Instructions in the file direct how to install the hook in your local Git environment (it won't run without this).
```

Comment:
We'll need to update this, I'd consider dropping the config validation on pre-commit. We don't check valid config in Calypso right now, most folks won't have the `circleci` available locally, and we don't want to slow down other commits.

Reply:
Good call. I have removed the pre-commits, and the mention of it in docs